kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -495,7 +496,7 @@ define([
                         });
 
                     //First, we find the expansion caret in the first cell.
-                    const $caret = $('td:eq(1)', row).find('i[data-caret]');
+                    const $caret = $('td:eq(1)', row).find('[data-caret]');
```

Comment:
This is the little carets next to the file icon; you click the caret to show file info. The carets are tiny, though -- at some point in the future, the click area should be expanded so you don't have pixel-hunt to see file metadata!